content/sdk/10.js/00.ethers/20.v6/00.getting-started.md

```diff
@@ -1,69 +1,154 @@
 ---
 title: "Getting Started"
 description: "Explore zkSync-ethers SDK version 6"
-tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "javascript", "v6"]
+tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "javascript", v6"]
 ---
 
-While most of the existing SDKs should work out of the box, deploying smart contracts or using unique zkSync features,
-like account abstraction, requires providing additional fields to those that Ethereum transactions have by default.
+## Overview
 
-To begin, it is useful to have a basic understanding of the types of objects available and what they are
-responsible for, at a high level:
+This guide will help you interact with the zkSync Era blockchain using the zkSync JavaScript SDK. We'll walk through
+creating a script that connects to the zkSync network, fetches network information, and retrieves transaction details.
 
-- `Provider` provides connection to the zkSync Era blockchain, which allows querying the blockchain state,
-  such as account, block or transaction details, querying event logs or evaluating read-only code using call.
-  Additionally, the client facilitates writing to the blockchain by sending transactions.
-- `Wallet` wraps all operations that interact with an account. An account generally has a private key, which
-  can be used to sign a variety of types of payloads. It provides easy usage of the most common features.
+## Key Concepts
 
-## Examples
+- **Provider**: A provider connects to the zkSync Era blockchain, allowing you to query details such as accounts, blocks,
+and transactions, and to send transactions to the blockchain.
+- **Wallet**: A wallet manages operations involving an account, typically using a private key for signing actions.
 
-The following examples could be easily run by writing the code snippets in a file and executing them as follows:
+## Prerequisites
 
-```shell
-npx ts-node src/zksync.ts
+Ensure you have the following installed:
+
+- Node.js
+- TypeScript
+- `zksync-ethers` library
+
+## Setup Instructions
+
+#### Step 1: Create a Scripts Folder
+
+In your project directory, create a folder named `scripts`:
+
+```bash
+mkdir scripts
 ```
 
-Connect to the zkSync Era network:
+#### Step 2: Create the `zksync.ts` File
 
-```ts
-import { Provider, utils, types } from "zksync-ethers";
-import { ethers } from "ethers";
+Inside the `scripts` folder, create a file named `zksync.ts`:
 
-const provider = Provider.getDefaultProvider(types.Network.Sepolia);
-const ethProvider = ethers.getDefaultProvider("sepolia");
+```bash
+touch scripts/zksync.ts
+```
```

Comment:
Can we remove the first step? Is there a reason to create a scripts folder? That adds an extra step that is unnecessary but makes it feel like it's a requirement to use this library. We don't know what their project setup looks like and shouldn't force them to take additional unnecessary steps.

Reply:
I've added an intro